content/docs/components-and-props.md

```diff
@@ -16,57 +16,59 @@ prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
 
-Components let you split the UI into independent, reusable pieces, and think about each piece in isolation. This page provides an introduction to the idea of components. You can find a [detailed component API reference here](/docs/react-component.html).
+Komponenter låter dig bryta upp användargränssnittet i oberoende, återanvändbara bitar och tänka på varje bit isolerat. Den här sidan ger dig en introduktion till tanken bakom komponenter. Du kan hitta ett [detaljerat API för komponenter här](/docs/react-component.html).
 
-Conceptually, components are like JavaScript functions. They accept arbitrary inputs (called "props") and return React elements describing what should appear on the screen.
+Konceptuellt är komponenter som JavaScript-funktioner. De accepterar godtyckliga inputs (kallade "props") och returnerar React-element som beskriver vad som ska visas på skärmen.
 
-## Function and Class Components {#function-and-class-components}
+## Funktion och klasskomponenter {#function-and-class-components}
```

Comment:
I would say this is more fitting as `Funktion- och klasskomponenter`, see https://sv.wikipedia.org/wiki/Berg-_och_dalbana

Reply:
@rloqvist Fixing. 👍